Dockerfile

```diff
@@ -58,3 +58,10 @@ RUN SWIFT_URL=https://swift.org/builds/$SWIFT_BRANCH/$(echo "$SWIFT_PLATFORM" |
 
 # Print Installed Swift Version
 RUN swift --version
+
+WORKDIR /code
+ONBUILD COPY Package.swift /code/
+ONBUILD RUN swift build -c release
```

Comment:
Also add `|| true` to the end of this. If you run swift build without Sources it sometimes returns a non-zero exit code, which fails the build. This command is really an attempt at caching dependency build products, but it's not required to succeed.